src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -54,6 +54,9 @@
     @Getter
     private static LogUnitServer logUnitServer;
 
+    @Getter
+    private static ManagementServer managementServer;
+
     public static boolean serverRunning_p = false;
```

Comment:
Yeah I can refactor the name to serverRunning.
It will require a refactor in ICmdlet too so I'll add that change to this PR